### ISSUE TITLE
find_package(charls 2.2.0 REQUIRED) not working

### ISSUE THREAD

**Issue by malaterre** [team-charls/charls#102]
Not clear if the issue is upstream or in Debian package. But:

```
cmake_minimum_required(VERSION 3.18)
project(p)
find_package(charls 2.2.0 REQUIRED)
```

lead to:

> CMake Error at CMakeLists.txt:3 (find_package):
>   Could not find a configuration file for package "charls" that is compatible
>   with requested version "2.2.0".
> 
>   The following configuration files were considered but not accepted:
> 
>     /usr/lib/x86_64-linux-gnu/cmake/charls/charlsConfig.cmake, version: unknown
>     /lib/x86_64-linux-gnu/cmake/charls/charlsConfig.cmake, version: unknown
> 
> 
> 
> -- Configuring incomplete, errors occurred!
> See also "/tmp/a/bin/CMakeFiles/CMakeOutput.log".
>

**Comment by malaterre** [comment]
For reference:

* https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=995831

**Comment by vbaderks** [comment]
Based on the provided cmake URL link, I think that the current CharLS CMake scripts miss some steps to create the correct package.

**Comment by vbaderks** [comment]
This should be resolved with 2e6e92dc3e88dc6e2553f84475cacff09f2eb8da
